components/brave_sync/brave_sync_p3a.h

```diff
@@ -14,6 +14,11 @@ namespace p3a {
 // TODO(alexeybarabash): move here also "Brave.Sync.Status.2" and
 // "Brave.Sync.ProgressTokenEverReset"
 inline constexpr char kEnabledTypesHistogramName[] = "Brave.Sync.EnabledTypes";
+inline constexpr char kSyncedObjectsCountHistogramNameV2[] =
+    "Brave.Sync.SyncedObjectsCount.2";
+
+// Obsolete metric name, with a new History datatype contains wrong data for
+// synced History objects count
```

Comment:
why not just remove it

Reply:
I thought about the case when the old name can appear and we want to understand what is that, but remove it seems better, thanks  https://github.com/brave/brave-core/pull/21623/commits/af2ea3411fda33ac4e1e9a954d812178d5ce30ae#diff-ba503643f47ecd99ba1e3736d0e78cd80c162471ba953bb99828926e5ef70757L20